Tempura/Navigation/NavigationActions.swift

```diff
@@ -161,46 +161,6 @@ public struct Hide: NavigationSideEffect {
   }
 }
 
-// MARK: - Katana Helpers
-
-extension AnyStore {
-  @available(*, deprecated, message: "Deprecated in favor of Katana's dispatch")
-  @discardableResult
-  public func dispatch<RSE: NavigationSideEffect>(_ dispatchable: RSE) -> Promise<Void> {
-    return self.anyDispatch(dispatchable).void
-  }
-
-  @available(*, deprecated)
-  public func awaitDispatch<RSE: NavigationSideEffect>(_ dispatchable: RSE) throws {
-    return try Hydra.await(self.dispatch(dispatchable))
-  }
-}
-
-extension AnySideEffectContext {
-  @available(*, deprecated, message: "Deprecated in favor of Katana's dispatch")
-  @discardableResult
-  public func dispatch<RSE: NavigationSideEffect>(_ dispatchable: RSE) -> Promise<Void> {
-    return self.anyDispatch(dispatchable).void
-  }
-
-  @available(*, deprecated)
-  public func awaitDispatch<RSE: NavigationSideEffect>(ramen dispatchable: RSE) throws {
-    return try Hydra.await(self.dispatch(dispatchable))
-  }
-}
-
-extension ViewController {
-  @discardableResult
-  public func __unsafeDispatch<RSE: NavigationSideEffect>(_ dispatchable: RSE) -> Promise<Void> {
-    return self.store.dispatch(dispatchable)
-  }
-
```

Comment:
Is this on purpose? this was not deprecated

Reply:
I think this should also have been deprecated

Reply:
It was completely useless anyway. There's an `__unsafeDispatch` that takes a `Dispatchable`, and `NavigationSideEffect` is a `Dispatchable`. So this overload was actually doing nothing.